### PR TITLE
Add meta tags

### PR DESCRIPTION
**Pull request by jackreimers**
Add meta tags to the page so that it is more easily googleable and provides better information to web crawlers.

Related to #85 

image
**Figure: Current result on Google**

image
**Figure: Meta tags on current production deployment**

image
**Figure: Meta tags on staging deployment**

### CODE REVIEW

**Review by Harry-Ross — APPROVED** [review]
lgtm

File: src/WebUI/wwwroot/index.html [hunk 1]
```diff
@@ -43,6 +43,14 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+
+    <meta Name="keywords" Content="GPT, ChatGPT, OpenAI, SSW, SSW Rules, Rule, Rules" />
```
Comment (bradystroud) [inline src/WebUI/wwwroot/index.html:47]: How many can we have?

I would add AI

File: src/WebUI/wwwroot/index.html [hunk 2]
```diff
@@ -43,6 +43,14 @@
...
+    <meta Property="og:title" Content="SSW RulesGPT" />
+    <meta Name="twitter:title" Content="SSW RulesGPT" />
+    <meta Name="description" Content="SSWBot is a helpful and informative GPT bot designed to assist users with rules related queries. " />
+    <meta Property="og:description" Content="SSWBot is a helpful and informative GPT bot designed to assist users with rules related queries. " />
+    <meta Name="twitter:description" Content="SSWBot is a helpful and informative GPT bot designed to assist users with rules related queries. go" />
```
Comment (bradystroud) [inline src/WebUI/wwwroot/index.html:52]:
```suggestion
    <meta Name="twitter:description" Content="SSWBot is a helpful and informative GPT bot designed to assist users with rules related queries." />
```

Was the `go` bit intentional?

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: src/WebUI/wwwroot/index.html (hunk 2 above)
Comment (jackreimers) [reply src/WebUI/wwwroot/index.html:52]: Negative

**Review by bradystroud — APPROVED** [review]

**Comment by bradystroud** [comment]
Looks like its working ✅

image


FYI I wouldn't close the issue until I know its working

**Pull request merged**